ksql-rest-app/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -277,7 +279,12 @@ public boolean hasNext() {
         String responseLine = responseScanner.nextLine().trim();
         if (!responseLine.isEmpty()) {
           try {
-            bufferedRow = objectMapper.readValue(responseLine, StreamedRow.class);
+            GenericRow genericRow = JsonUtil.buildGenericRowFromJson(responseLine);
```

Comment:
This is the forth time I'm flagging up that this breaks current functionality (and highlights a missing test). With this change the CLI won't output any error messages or "last message". The "last message" is used by LIMIT statements to send a "Limit reached" message back.

Would be great if you could add a test to catch this breaking change.

Reply:
Reverted this change to use StreamedRow as before.